CHANGES.rst

```diff
@@ -194,6 +194,9 @@ Bug Fixes
 
 - ``astropy.visualization``
 
+  - Avoid importing matplotlib.pyplot when importing
+    astropy.visualization.wcsaxes. [#5680]
```

Comment:
@astrofrog , as your bot stated, there is no reference to this PR here. Is this intentional?